### ISSUE TITLE
`context=module` must be quoted

### ISSUE THREAD

**Issue by mustafa0x** [sveltejs/vite-plugin-svelte#804]
### Describe the bug

```
[vite-plugin-svelte] Error while preprocessing ... If the context attribute is supplied, its value must be "module"
 16 |  
 17 |  <script context=module>
               ^
 18 |  import {writable} from 'svelte/store'
 19 |  
```

Quotes are now required, while they weren't.

### System Info

```shell
npmPackages:
    @sveltejs/vite-plugin-svelte: ^3.0.0 => 3.0.0 
    svelte: ^4.2.5 => 4.2.5 
    vite: ^5.0.0 => 5.0.0 
```

**Comment by mustafa0x** [comment]
`<script context=module>` is found ~50 times on github.

https://github.com/search?q=language%3Asvelte+%2F%3Cscript+context%3Dmodule%3E%2F&type=code

**Comment by paulocoghi** [comment]
This is now fixed in Svelte v4.2.6, with PR https://github.com/sveltejs/svelte/pull/9551
